kernel/arch/dreamcast/hardware/maple/controller.c

```diff
@@ -8,11 +8,15 @@
 #include <arch/arch.h>
 #include <dc/maple.h>
 #include <dc/maple/controller.h>
-#include <string.h>
+#include <kos/mutex.h>
+#include <kos/worker_thread.h>
```

Comment:
I wonder if we can use standard C17 threads instead of KOS threads?

Reply:
This is inside KOS though, so it makes sense to use KOS threads.

C11/C17 threads are implemented on top of KOS' threads so it would be adding a layer of indirection for no real reason.

Reply:
Yeah, we are the ones providing C17 threads... using KOS threads.